src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -77,44 +96,78 @@ const ButtonWrapper = styled.div`
   align-items: center;
 
   @media (max-width: ${(props) => props.theme.breakpoints.md}) {
-    position: sticky;
+    position: fixed;
     bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100vw;
+    box-sizing: border-box;
     background: ${(props) => props.theme.colors.background};
-    padding: ${(props) => props.theme.space.md};
+    padding: ${(props) => props.theme.space.lg};
     border: none;
     border-top: ${(props) => props.theme.borders.sm}
       ${(props) => getColorV8("grey", 300, props.theme)};
   }
 `;
-interface ServiceCatalogItemPageProps {
+
+export interface ServiceCatalogItemPageProps {
   serviceCatalogItem: ServiceCatalogItem;
+  baseLocale: string;
+  hasAtMentions: boolean;
+  userRole: string;
+  userId: number;
+  organizations: Array<Organization>;
 }
 
 export function ServiceCatalogItemPage({
   serviceCatalogItem,
+  baseLocale,
+  hasAtMentions,
+  userRole,
+  organizations,
+  userId,
 }: ServiceCatalogItemPageProps) {
   const [isExpanded, setIsExpanded] = useState<boolean>(false);
+  const { requestFields, handleChange } = useItemFormFields(serviceCatalogItem);
   const { t } = useTranslation();
 
+  const defaultOrganizationId =
+    organizations.length > 0 && organizations[0]?.id
+      ? organizations[0]?.id?.toString()
+      : null;
+
   const toggleDescription = () => {
     setIsExpanded(!isExpanded);
   };
 
   return (
     <Container>
       <LeftColumn>
-        <ItemTitle tag="h1">{serviceCatalogItem.name}</ItemTitle>
-        <CollapsibleDescription expanded={isExpanded}>
-          {serviceCatalogItem.description}
-        </CollapsibleDescription>
-        <ToggleButton aria-hidden="true" isLink onClick={toggleDescription}>
-          {isExpanded
-            ? t("service-catalog.item.read-less", "Read less")
-            : t("service-catalog.item.read-more", "Read more")}
-          <Button.EndIcon>
-            {isExpanded ? <ChevronUp /> : <ChevronDown />}
-          </Button.EndIcon>
-        </ToggleButton>
+        <DescriptionWrapper>
+          <ItemTitle tag="h1">{serviceCatalogItem.name}</ItemTitle>
+          <CollapsibleDescription expanded={isExpanded}>
+            {serviceCatalogItem.description}
+          </CollapsibleDescription>
+          <ToggleButton aria-hidden="true" isLink onClick={toggleDescription}>
```

Comment:
As discussed, we need to remove this `aria-hidden` attribute since it is not compliant